source/cmake/FindROCM.cmake

```diff
@@ -7,11 +7,21 @@
 # ROCM_LIBRARIES
 
 # define the search path
-list(APPEND ROCM_search_PATHS ${ROCM_ROOT})
-list(APPEND ROCM_search_PATHS "/opt/rocm/")
+if(ROCM_ROOT)
+  list(APPEND ROCM_SEARCH_PATHS ${ROCM_ROOT})
+else()
+  if(ROCM_DTK)
+    list(APPEND ROCM_SEARCH_PATHS "/public/software/compiler/dtk/dtk-21.04/")
+    list(APPEND ROCM_SEARCH_PATHS "/public/software/compiler/rocm/dtk-21.04/")
```

Comment:
I think these two paths should be different in different machines?

Reply:
yes, These two paths are the rocm_dtk paths that I have seen. I put them here as the default paths if user does not give the path of ROCm. And if the ROCm_dtk's path is not in the default path-lists, he/she can use `ROCM_ROOT` to attach to path

Reply:
yes, These two paths are the rocm_dtk paths that I have seen. I put them here as the default paths if user does not give the path of ROCm. And if the ROCm_dtk's path is not in the default path-lists, user can use ROCM_ROOT to attach to path

Reply:
Is it the default installation path? It does not look like so.

Reply:
As a matter of fact, it is the default installation path.
On the cluster, '/opt/rocm' maybe looks like the default path. But this is the path of the version 2.9